core/templates/core/user_accounts/account_edit.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'core/user_accounts/account_base.html' %}
```

Comment:
This file is identical to the other `account_edit.html` file created in this PR. Feel free to remove this one as it's not actually used.